src/commonmark/en/content/sysadmin/audit.md

```diff
@@ -0,0 +1,48 @@
+# Audit
+
+## Introduction
+
+DHIS2 supports a new audit service which is based on Apache ActiveMQ Artemis. Artemis is used as an asynchronous messaging system by DHIS2.
+
+Aafter an entity is saved to database, an audit message will be sent to the Artemis message consumer service. The message will then be processed in a different thread.
+
+Audit logs can be retrieved from the DHIS2 database. Currently there is no UI or API endpoint available for retriving audit entries.
+
+Note that by default, the auditing service is disabled.
+
+
+## Single Audit table
+
+All audit entries will be saved into one single table named `audit`.
+
+| Column     | Type                        | Description   |
+|------------|-----------------------------|---------------|
+| auditid    | integer                     |               |
+| audittype  | text                        | READ, CREATE, UPDATE, DELETE, SEARCH |
+| auditscope | text                        | METADATA, AGGREGATE, TRACKER |
+| klass      | text                        | Audit Entity Java class name |
+| attributes | jsonb                       | Json string stores attributes of the audit entity, used for searching. Example: {"valueType":"TEXT", "categoryCombo":"SWQW313FQY", "domainType":"TRACKER"} |   |
+| data       | bytea                       | Compressed Json string of the Audit Entity    
+| createdat  | timestamp without time zone |               |
+| createdby  | text                        |               | 
+| uid        | text                        |               |
+| code       | text                        |               |
+
+## Setup
+
+The audit matrix configuration can be defined in `dhis.conf`  file as below. By default, audit is disabled.
+ 
+```properties
+metadata.audit.persist = on
+tracker.audit.persist = on
+audit.metadata = CREATE;UPDATE;DELETE
+audit.tracker = READ;CREATE;UPDATE;DELETE
```

Comment:
I would expand a bit on the configuration:

The audit system is automatically configured to audit for the following scopes and types:
- `CREATE`, `UPDATE`, `DELETE`
- `METADATA`, `TRACKER`, `AGGREGATE`

No action is required to activate the audit.
The audit can still be configured using the "audit matrix". The audit matrix is driven by 3 properties in `dhis.conf`:

- `audit.metadata`
- `audit.tracker`
- `audit.aggregate`

Each property accepts a semicolon delimited list of valid Audit Types:

- `CREATE`
- `UPDATE`
- `DELETE`
- `READ`

For instance, in order to only audit Tracker related object creation and deletion, the following property should be added  to `dhis.conf`:

`audit.tracker = CREATE;UPDATE`

In order to completely disable auditing, this is the configuration to use:

`audit.metadata = `
`audit.tracker = `
`audit.aggregate = `